views/014_job_history_status.sql

```diff
@@ -0,0 +1,60 @@
+CREATE
+OR REPLACE VIEW job_history_latest_status AS
+SELECT
+  jh.*
+FROM
+  job_history jh
+  JOIN (
+    SELECT
+      resource_id,
+      MAX(created_at) AS max_created_at
+    FROM
+      job_history
+    GROUP BY
+      resource_id
+  ) jh2 ON jh.resource_id = jh2.resource_id
+  AND jh.created_at = jh2.max_created_at;
+
+-- Template View
+CREATE
+OR REPLACE VIEW templates_with_status AS
+SELECT
+  t.*,
+  jhls.created_at job_created_at,
+  jhls.status job_status
+FROM
+  templates t
+  LEFT JOIN job_history_latest_status jhls ON t.id :: TEXT = jhls.resource_id;
```

Comment:
I think we need to filter on resource type as well

Reply:
I see that those are just labels given to the job. Couldn't find one for config scraper